lib/index.ts

```diff
@@ -751,7 +751,7 @@ export class Worker {
 		options: {
 			timestamp?: any;
 			reason?: any;
-			actor?: any;
+			actor: any;
```

Comment:
I suspect that we're not currently setting `actor` in a number of places when calling `worker.insertCard()`. Let's take this PRs beta package and run it through the CI for plugins and the main repo. Also requiring an option that was previously optional is a breaking change, so will need to update the `Change-type` to `major`.